src/app/Instance/BridgeOverview/BridgeOverview.test.tsx

```diff
@@ -0,0 +1,130 @@
+import { ManagedResourceStatus } from "@rhoas/smart-events-management-sdk";
+import { fireEvent, RenderResult } from "@testing-library/react";
+import { customRender, waitForI18n } from "@utils/testUtils";
+import React from "react";
+import { BrowserRouter } from "react-router-dom";
+import { BridgeOverview, BridgeOverviewProps } from "./BridgeOverview";
+import { processorData } from "./BOStoriesHelper";
+
+const IngressEndpoint =
+  "https://3543edaa-1851-4ad7-96be-ebde7d20d717.apps.openbridge-dev.fdvn.p1.openshiftapps.com/events";
+
+const setupBridgeOverview = (
+  props: Partial<BridgeOverviewProps>
+): { comp: RenderResult } => {
+  const {
+    onCreateProcessor = jest.fn(),
+    instanceId = "3543edaa-1851-4ad7-96be-ebde7d20d717",
+    processorList,
+    bridgeStatus,
+    processorsError,
+    bridgeIngressEndpoint,
+  } = props;
+  const comp = customRender(
+    <BrowserRouter>
+      <BridgeOverview
+        onCreateProcessor={onCreateProcessor}
+        instanceId={instanceId}
+        processorList={processorList}
+        processorsError={processorsError}
+        bridgeStatus={bridgeStatus}
+        bridgeIngressEndpoint={bridgeIngressEndpoint}
+      />
+    </BrowserRouter>
+  );
+  return { comp };
+};
+
+describe("Bridge Overview", () => {
+  it("should check for getting started card is open by default", async () => {
+    const { comp } = setupBridgeOverview({});
+    await waitForI18n(comp);
+    expect(comp.queryByText("Learn about YAML templates")).toBeInTheDocument();
+  });
+
+  it("should check for getting started card is closed, after clicking on toggle button", async () => {
+    const { comp } = setupBridgeOverview({});
+    await waitForI18n(comp);
+    expect(comp.getByLabelText("getting started toggler")).toBeInTheDocument();
+    fireEvent.click(comp.getByLabelText("getting started toggler"));
+    expect(
+      comp.queryByText("Learn about YAML templates")
+    ).not.toBeInTheDocument();
+  });
+
+  it("should have ingress endpoint", async () => {
+    const { comp } = setupBridgeOverview({
+      bridgeIngressEndpoint: IngressEndpoint,
+    });
+    await waitForI18n(comp);
+
+    expect(comp.getByDisplayValue(IngressEndpoint)).toBeInTheDocument();
+  });
+
+  it("should display no processor", async () => {
+    const { comp } = setupBridgeOverview({ processorList: [] });
+
+    await waitForI18n(comp);
+
+    expect(comp.getByText("No processors")).toBeInTheDocument();
+  });
+
+  it("should display list of processors", async () => {
+    const { comp } = setupBridgeOverview({
+      processorList: processorData,
+    });
+
+    await waitForI18n(comp);
+
+    expect(comp.queryByText("No processors")).not.toBeInTheDocument();
+    expect(comp.queryByText("Processor one")).toBeInTheDocument();
+    expect(comp.queryByText("Processor two")).toBeInTheDocument();
+    expect(comp.queryByText("Processor three")).toBeInTheDocument();
+    expect(comp.queryByText("Processor four")).toBeInTheDocument();
```

Comment:
You can loop through the data instead of manually check each one:

```suggestion
    processorData.map((processor) => {
      expect(comp.queryByText(processor.name)).toBeInTheDocument();
    });
```

You can eventually do the same to check the href's of all the links.